Source/ACE.Server/Managers/WorldManager.cs

```diff
@@ -451,7 +451,7 @@ private static void DoPlayerEnterWorld(Session session, Character character, Pla
             if (character.TotalLogins <= 1 || PropertyManager.GetBool("alwaysshowwelcome").Item)
             {
                 // check the value of the welcome message. Only display it if it is not empty
-                string welcomeHeader = ConfigManager.Config.Server.Welcome ?? "Welcome to Asheron's Call!";
+                string welcomeHeader = ConfigManager.Config.Server.Welcome != "" ? ConfigManager.Config.Server.Welcome : "Welcome to Asheron's Call!";
```

Comment:
If you're going to change from null to "" check, it should be a String.IsEmpty() check.